cmd/kops/update_cluster.go

```diff
@@ -109,6 +113,8 @@ func NewCmdUpdateCluster(f *util.Factory, out io.Writer) *cobra.Command {
 	cmd.Flags().StringVar(&options.OutDir, "out", options.OutDir, "Path to write any local output")
 	cmd.Flags().BoolVar(&options.CreateKubecfg, "create-kube-config", options.CreateKubecfg, "Will control automatically creating the kube config file on your local filesystem")
 	cmd.Flags().StringVar(&options.Phase, "phase", options.Phase, "Subset of tasks to run: "+strings.Join(cloudup.Phases.List(), ", "))
+	cmd.Flags().StringSliceVar(&options.LifecycleOverrides, "lifecycle-overrides", options.LifecycleOverrides, "comma separated list of phase overrides, example: SecurityGroups=Ignore,InternetGateway=ExistsAndWarnIfChanges")
```

Comment:
I separated this out so that a phase does not have to be set.  If we include this with the `--phase` flag a user would not be able to run all of the phases.  Thoughts?

Reply:
I am open to other ideas, but I do not want to force a user to define a phase in order to use overrides.  Users will want to use overrides but not specify a phase.  We can do phase="", but that is ugly and a change to syntax.  Phases are a group of tasks and lifecycles, while lifecycle overrides are just setting the lifecycle on a single task.  Kinda stumped how to do this well.

@rdrgmnzs I know you are going to use this functionality, any opinion?